code/modules/asset_cache/assets/tcomms.dm

```diff
@@ -0,0 +1,14 @@
+/datum/asset/spritesheet/telecomms
+	name="tcomms"
+
+/datum/asset/spritesheet/telecomms/create_spritesheets()
+	var/list/inserted_states = list() // No need to send entire `telecomms.dmi`.
+	for(var/path as anything in subtypesof(/obj/machinery/telecomms))
+		var/obj/machinery/telecomms/machine = path
```

Comment:
```suggestion
	for(var/obj/machinery/telecomms/machine as anything in subtypesof(/obj/machinery/telecomms))
```
Do this or remove the ``as anything``, since it's meant to remove checking for type it's not necessary to have if there's no type to check in the first place